curricula/tests/test_user_permissions.py

```diff
@@ -0,0 +1,353 @@
+from django.test import TestCase
+
+from django.contrib.auth.models import Permission
+
+from lessons.factories import LessonFactory
+from curricula.factories import CurriculumFactory, UnitFactory, ChapterFactory, UserFactory, GroupFactory
+
+# Tests for admin pages in lessons/admin.py
+class UserPermissionsTestCase(TestCase):
+
+    def setUp(self):
+        # Create user groups with permissions
+        author_group = self.create_author_group()
+
+        partner_group = self.create_partner_group()
+
+        # Set up users
+        partner_user = UserFactory()
+        partner_user.is_staff = True
+        partner_group.user_set.add(partner_user)
+        partner_user.save()
+        self.partner_user = partner_user
+        self.assertTrue(self.client.login(username=partner_user.username, password='password'))
+
+        author_user = UserFactory()
+        author_user.is_staff = True
+        author_group.user_set.add(author_user)
+        author_user.save()
+        self.author_user = author_user
+        self.assertTrue(self.client.login(username=author_user.username, password='password'))
+
+        # Create curriculum, unit, chapter, lesson
+        self.partner_curriculum = CurriculumFactory(user=partner_user, slug="partner-curriculum")
+        self.partner_unit = UnitFactory(user=partner_user, parent=self.partner_curriculum, slug="partner-unit")
+        self.partner_chapter = ChapterFactory(user=partner_user, parent=self.partner_unit, slug="partner-chapter")
```

Comment:
if you don't need to pass in the urls below, then you won't need to set the slugs on these factory calls either.